features/providers/check_provider_answers.feature

```diff
@@ -26,6 +28,8 @@ Feature: Checking client details answers backwards and forwards
       | question | answer |
       | First name | Test |
       | Last name | Walker |
+      | Has your client ever changed their last name? | No |
+      | What was your client's last name at birth? | Walker |
```

Comment:
Is this line being displayed if the answer to "Has your client ever changed their last name?" is no?

Reply:
Yes @agoldstone93 I thought in this instance it should display the last name. Do you think that is wrong/should be changed?

Reply:
👍 I have just checked with Sara and she says it wouldn't be displayed so will update